test/actions/katello/repository/discover_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'katello_test_helper'
+
+module Actions
+  describe Katello::Repository::CloneToVersion do
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    let(:action_class) { ::Actions::Katello::Repository::Discover }
+
+    def setup
+      get_organization #ensure we have an org label
+    end
+
+    def test_discovers_without_encryption
+      EncryptionKey.const_set(:ENCRYPTION_KEY, nil)
```

Comment:
I didn't have an encryption key in my dev env so this broke.

Maybe better to mock it out completely:

```
module ::EncryptionKey                                                                                                                                                         
  ENCRYPTION_KEY = nil                                                                                                      
end
```

similar for the following test

Reply:
Thanks, updated!